MAUI/DataGrid/Column-sizing.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: Column Sizing in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Column Sizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more.
+platform: maui
+control: SfDataGrid
+documentation: ug
+---
+
+# Column Sizing in .NET MAUI DataGrid (SfDataGrid)
+
+DataGrid allows you to set the column widths based on certain logic using `SfDataGrid.ColumnWidthMode` or `DataGridColumn.ColumnWidthMode`property. Below is the list of predefined column sizing options available.
+
+<table>
+<tr>
+<th>
+Type
+</th>
+<th>
+Column width
+</th>
+</tr>
+<tr>
+<td>
+<code>Fill</code>
+</td>
+<td>
+Divides the total width equally for columns. 
+</td>
+</tr>
+<tr>
+<td>
+<code>Auto</code>
+</td>
+<td>
+Calculates the width of column based on header and cell contents. So that header and cell content’s are not truncated.  
+</td>
+</tr>
+<tr>
+<td>
+<code>LastColumnFill</code>
+</td>
+<td>
+The column width of the DataGridColumns are adjusted with respect to SfDataGrid.DefaultColumnWidth property. In case the columns does not fill the entire view space, width of the last column fills the unoccupied space in the view.
+</td>
+</tr>
+<tr>
+<td>
+<code>FitByCell</code>
+</td>
+<td>
+Calculates the width of column based on cell contents. So that cell content’s are not truncated.  
```

Comment:
Calculates the width of columns based on cell contents so that cell contents are not truncated.